src/metabase/models/params.clj

```diff
@@ -176,11 +176,11 @@
 ;;; |                                               DASHBOARD-SPECIFIC                                               |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
-(s/defn ^:private dashboard->parameter-mapping-field-clauses :- (s/maybe #{mbql.s/field})
+(s/defn ^:private dashcards->parameter-mapping-field-clauses :- (s/maybe #{mbql.s/field})
```

Comment:
This change allows optimization of DB calls because the dashboard isn't required to be selected and hydrated. Only its dashcards are needed.